2.1.0 Doc Changes

cs-abhishek-shukla opened the pull request:
- Rewrote README.md
- Rewrote setup.md
- Rewrote configuration instructions and usage
- Fixed, typo, structure, and other information flow issues

cs-yash-patodkar commented on docs/res/.DS_Store:
Please remove this file

cs-dev-chaudhary commented on docs/res/soc_mgmt_edit_json.png:
Please replace the screenshot attached with comment. As we have changed the field name from second 'data source' to 'Json Data Source Modules'. 
![Uploading MicrosoftTeams-image (11).png…]()

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -1,26 +1,47 @@
+
+| Fields                                     | Description                                                                                                                                               |
+|--------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Data Source                                | Select the module from which to fetch the data                                                                                                            |
```
'Data Source' is renamed to 'Json Data Source Modules'.

cs-yash-patodkar commented on docs/res/soc_management_light.png:
Need to change light and dark image, since the incident count doesnt match

cs-dev-chaudhary commented on docs/usage.md:
```diff
@@ -1,257 +1,358 @@
+    },
+    {
+      "id": "resolvedAutomatedly",
+      "title": "by Automation )",
```
Please replace "by automation" to "automatedly".

cs-abhishek-shukla committed:
2.1.0 Doc Changes
- Rewrote README.md
- Rewrote setup.md
- Rewrote configuration instructions and usage
- Fixed, typo, structure, and other information flow issues
- Changed sample json and images
- Edited the updated field name
- Updated the screenshot with the new field
- Annotated the images for each JSON key
